README.md

```diff
@@ -49,3 +49,28 @@ wish to make a clarifying public statement as follows:
 > incorporate enhancements is considered a derivative work of the product. Software that merely uses or aggregates (i.e.
 > links to) an otherwise unmodified version of existing software is not considered a derivative work, and therefore it
 > does not need to be released as under the same license, or even released as open source.
+
+
+## User & Programmers Manual
```

Comment:
I think README.md file is not the right place to include this. In addition, it should be simplified: you have already included the needed packages in packages.json and so on, so please focus just in the usage.

In sum, It should be included in development.md, just after the last item (Prettify Code) and be like this:

```
### Swagger

In order to run Swagger, you need to execute the Perseo FE (as explained [here](deployment.md) and then you
can access to:

```
<server_host>:9090/api-docs
```

The swagger documentation provided at /api-docs covers all the HTTP endpoint exposed by Perseo FE.
```

Reply:
Fixed in 210813ba445911619199c59d729c8b724f5aef5a